docs/content/preview/yugabyte-platform/manage-deployments/high-availability.md

```diff
@@ -126,22 +124,22 @@ Even though you can perform an upgrade of all YugabyteDB Anywhere instances simu
 The following is the detailed upgrade procedure:
 
 1. Stop the high-availability synchronization. This ensures that only backups of the original YugabyteDB Anywhere version are synchronized to the standby instance.
-2. Upgrade the active instance. Expect a momentary lapse in availability for the duration of the upgrade. If the upgrade is successful, proceed to step 3. If the upgrade fails, perform the following:
+1. Upgrade the active instance. Expect a momentary lapse in availability for the duration of the upgrade. If the upgrade is successful, proceed to step 3. If the upgrade fails, perform the following:
```

Comment:
2?

Reply:
Or it will auto number?

Reply:
Using 1. for all steps engages auto numbering, yes